### ISSUE TITLE
chunking & other modifications as the duration grows

### ISSUE THREAD

**Issue by wibeasley** [andkov/project-agrus#8]
> I guess we have a few options for leveraging the config file. One is to pull out the graph in a function that takes the language/config as a parameter. Another is each html file has its own language, which probably means we use [rmarkdown's report parameterization](https://bookdown.org/yihui/rmarkdown/parameterized-reports.html).
> 
> @andkov, once you specify the English title & subtitle, I think this issue can be closed. I just did some brainless text alignment, so make sure you pull those changes or the merge conflicts will be tedious.
> 
> ```yaml
>   title   : "**Title**"
>   subtitle: "**Subtitle**"
> ```
> 
> I'm thinking of other languages we can recruit friend for. I can easily get native speakers of Chinese, Spanish, French, & Norwegian. I'm sure you could get (or do yourself) languages close to the region like Polish & Russian. It probably makes sense to wait until the graph is finished maturing.

With respect to maturation, I think we'll need to start chunking up the graph as the number of days grows.  Maybe 30 days per canvas? Maybe 60?  Then, it will be a sequence of graphs, so a dedicated HTML will serve as an output, one per language.  But again, you are right = the design will be dictated by the total number of days, so we may need to adjust as we go along.  Hope we won't get to deal with that.

_Originally posted by @andkov in https://github.com/andkov/project-agrus/issues/7#issuecomment-1091946357_

**Comment by andkov** [comment]
@wibeasley , I agree, there should be that (k + 1)nth graph that shows the overall picture, with a simplified/adjusted labelling to suite the purpose. But I like (so far) the idea of having unique annotations to the months/canvases to emphasize their uniqueness.
